tests/components/hlk_sw16/test_config_flow.py

```diff
@@ -0,0 +1,82 @@
+"""Test the Hi-Link HLK-SW16 config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.hlk_sw16.config_flow import (
+    AlreadyConfigured,
+    CannotConnect,
+)
+from homeassistant.components.hlk_sw16.const import DOMAIN
+
+from tests.async_mock import patch
+
+hlk_sw16_test_config = {
+    "host": "1.1.1.1",
+    "port": 8080,
+}
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.hlk_sw16.config_flow.validate_input",
```

Comment:
Please patch the external library here instead of patching `validate_input`

Reply:
removed patch entirely